netlify.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+  command   = "npm run build"
```

Comment:
Does it have to be an `npm` script? Can ewe use `yarn build` instead as we're a yarn repo?

Reply:
We absolutely can, should, and will :)